### ISSUE TITLE
broken @lerepo/detect-package-manager package

### ISSUE THREAD

**Issue by magicdawn** [lerepo/workspace-tools#1]
![image](https://user-images.githubusercontent.com/4067115/200588813-e3103b8f-46aa-41d2-8011-89d1e918e640.png)

Node.js supports async function since v7.6.0 (released long long ago).
So there is no need to transpile async function.

**Comment by abdes** [comment]
You are right. 

However, `@lerepo/detect-package-manager` is replaced with `@lerepo/detect-pm` which is cleaner, more robust, newer and is the maintained version.

Would you consider replacing your dependency on the old package with the newer one?

```
> npm install @lerepo/detect-pm

added 6 packages, and audited 7 packages in 6s

found 0 vulnerabilities

>node
Welcome to Node.js v16.17.1.
Type ".help" for more information.
> require('@lerepo/detect-pm')
{
  detectInstalledPackageManagers: [Getter],
  detectPackageManagers: [Getter]
}
```

**Comment by magicdawn** [comment]
Thanks.

**Comment by magicdawn** [comment]
just found the @lerepo/detect-pm@0.3.1 package did not ship d.ts file, and no type field in package.json, but the old detect-package-manager did.

I see your workflow has script entry `yarn t:types`, but the released version just missing that.

https://unpkg.com/browse/@lerepo/detect-pm@0.3.1/package.json

**Comment by abdes** [comment]
You are correct again :-)

The reason is that I did not pay attention to where the output of `tsc` declaration files was going. It was going to the `src` folder inside `lib`. It should go directly under `lib`. 

I'm gonna correct that for all packages in the repo and re-releasse.

This is what the package contents will look like:

```
./CHANGELOG.md
./lib
./lib/detect-installed.d.ts
./lib/detect-installed.js
./lib/detect.d.ts
./lib/detect.js
./lib/index.d.ts
./lib/index.js
./lib/internals
./lib/internals/find-pm.d.ts
./lib/internals/find-pm.js
./lib/internals/has-pm.d.ts
./lib/internals/has-pm.js
./lib/internals/manifest.d.ts
./lib/internals/manifest.js
./lib/internals/utils.d.ts
./lib/internals/utils.js
./lib/package-manager.d.ts
./lib/package-manager.js
./lib/types
./lib/types/declaration.d.js
./LICENSE
./package.json
./README.md

```

**Comment by abdes** [comment]
Fixed with version 0.3.3
